clap-v3-utils/src/input_parsers/mod.rs

```diff
@@ -0,0 +1,477 @@
+use {
+    crate::input_validators::normalize_to_url_if_moniker,
+    chrono::DateTime,
+    clap::ArgMatches,
+    solana_sdk::{
+        clock::UnixTimestamp, commitment_config::CommitmentConfig, genesis_config::ClusterType,
+        native_token::sol_to_lamports, pubkey::MAX_SEED_LEN,
+    },
+    std::str::FromStr,
+};
+
+pub mod signer;
+#[deprecated(
+    since = "1.17.0",
+    note = "Please use the functions in `solana_clap_v3_utils::input_parsers::signer` directly instead"
+)]
+pub use signer::{
+    keypair_of, keypairs_of, pubkey_of, pubkey_of_signer, pubkeys_of, pubkeys_of_multiple_signers,
+    pubkeys_sigs_of, resolve_signer, signer_of, STDOUT_OUTFILE_TOKEN,
+};
```

Comment:
When you create the SignerSource parsers, you're going to deprecate all these methods anyway, right?
Does it maybe make sense to just leave them where they are for now so no one starts depending on the new `solana_clap_v3_utils::input_parsers::signer` paths that we're about to deprecate?

Reply:
Yeah, so the functions that panic: `keypair_of`, `keypairs_of`, `pubkey_of`, and `pubkeys_of` will be deprecated in favor of the try versions of these. The others that return a generic error type will not panic, so we should be able to use them. Clap does not support multi-argument validation/parsing, so we will need to keep the interfaces for these functions anyway.

I returned the functions that will be deprecated back to `mod.rs`, but let me know what you think. I can also just undo the refactoring and just have everything in the same file 😅 as well to just make things simpler.

Reply:
This works for me! Thanks